src/core/labeling/qgslabelfeature.h

```diff
@@ -471,6 +470,26 @@ class CORE_EXPORT QgsLabelFeature
      */
     void setLineAnchorType( QgsLabelLineSettings::AnchorType type ) { mLineAnchorType = type; }
 
+    /**
+     * Returns the line anchor text point, which dictates which part of the label text
```

Comment:
Does it need to be as this, mimicking the next function?
```suggestion
     * Returns the line anchor text \a point, which dictates which part of the label text
```